index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item header-item1"><time id="time">9:41</time></div>
+    <div class="header-item header-item2"><img id="signal" src="src/signal.png" alt="signal"></div>
+    <div class="header-item header-item3"><p id="name">Anand Pathak</p></div>
```

Comment:
I suggest you change this from a `div` to a link element. 
Logo are usually links that open  the homepage. For this milestone, you only have one page however please make this a link. You can read more on this [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)